test/test-sets/ref/ml100k_small.stderr

```diff
@@ -9,17 +9,17 @@ Reading datafile = test-sets/ml100k_small_test
 num sources = 1
 average  since         example        example  current  current  current
 loss     last          counter         weight    label  predict features
-2.655687 2.655687            1            1.0   5.0000   3.3704       23
-2.862807 3.069926            2            2.0   5.0000   3.2479       23
-2.392509 1.922212            4            4.0   4.0000   3.2218       23
-1.468514 0.544518            8            8.0   4.0000   3.2538       23
-1.515712 1.562910           16           16.0   5.0000   3.1581       23
+2.491066 2.491066            1            1.0   5.0000   3.4217       23
+2.698820 2.906574            2            2.0   5.0000   3.2951       23
+2.381092 2.063364            4            4.0   4.0000   3.2078       23
+1.469532 0.557971            8            8.0   4.0000   3.2111       23
+1.509373 1.549214           16           16.0   5.0000   3.2134       23
 
 finished run
 number of examples per pass = 30
 passes used = 1
 weighted example sum = 30.000000
 weighted label sum = 110.000000
-average loss = 1.471805
+average loss = 1.457823
```

Comment:
It's rather odd that the new way of doing random initialization seems to have helped in every test case.